### PR TITLE
Package containers.2.0+alpha1

### PR DESCRIPTION
**Pull request by c-cube**
### `containers.2.0+alpha1`

A modular, clean and powerful extension of the OCaml standard library.

Containers is an extension of OCaml's standard library (under BSD license)
focused on data structures, combinators and iterators, without dependencies on
unix, str or num. Every module is independent and is prefixed with 'CC' in the
global namespace. Some modules extend the stdlib (e.g. CCList provides safe
map/fold_right/append, and additional functions on lists).
Alternatively, `open Containers` will bring enhanced versions of the standard
modules into scope.

It also features sub-libraries for dealing with threads, S-expressions,
and the intricacies of unix.

---
* Homepage: https://github.com/c-cube/ocaml-containers/
* Source repo: https://github.com/c-cube/ocaml-containers.git
* Bug tracker: https://github.com/c-cube/ocaml-containers/issues/

---

:camel: Pull-request generated by opam-publish v0.3.5

### CODE REVIEW

**Comment by c-cube** [comment]
Note to opam-repo maintainers: there are some packages with no explicit upper bound on containers that are going to need one (< 2.0) soon.

**Comment by jpdeplaix** [comment]
Yes, revdeps are breaks a lot of packages. Could you:
* Restrict ```calculon```,  ```nsq```, ```oth```, ```smbc``` and ```snabela``` to < 2.0
* Restrict ```logtk``` and ```zipperposition``` to ```sequence < 1.0``` (seem to have been forgotten) and then try with containers 2.0+alpha1
* Try locally ```websocket``` as it seems that the CI cannot build it. I'll investigate.

**Comment by jpdeplaix** [comment]
I think the problem with ```websocket``` is that opam-depext is executing:
```opam list --external="x86_64,linux,debian"```
but it seems to be forbidden by opam master and the supported syntax seems to be:
```opam list --external "x86_64,linux,debian"```

cc @AltGr

File: packages/containers/containers.2.0+alpha1/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
+opam-version: "1.2"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "Simon Cruanes"
+homepage: "https://github.com/c-cube/ocaml-containers/"
+bug-reports: "https://github.com/c-cube/ocaml-containers/issues/"
+doc: "https://c-cube.github.io/ocaml-containers"
+tags: ["stdlib" "containers" "iterators" "list" "heap" "queue"]
+dev-repo: "https://github.com/c-cube/ocaml-containers.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-test: [make "test"]
```
Comment (rgrinberg) [inline packages/containers/containers.2.0+alpha1/opam]: Is it possible to use a jbuilder command to run the tests? There's no need to rely on make unless it's totally necessary for this.

Comment (c-cube) [reply packages/containers/containers.2.0+alpha1/opam]: it's on master, but can we please not block the PR for this?

**Comment by jpdeplaix** [comment]
Ok I was waiting for the MacOS CI to merge but it seems it's not coming any time soon.
Merging.

**Pull request merged**